packages/modules/backup_clouds/samba/backup_cloud.py

```diff
@@ -17,8 +17,12 @@ def upload_backup(config: SambaBackupCloudConfiguration, backup_filename: str, b
     if conn.connect(config.smb_server, 139):
         log.info("SMB Verbindungsaufbau erfolgreich")
         full_file_path = config.smb_path + backup_filename if config.smb_path is not None else backup_filename
-        log.info("Backup nach //" + config.smb_server + '/' + config.smb_share + full_file_path)
-        conn.storeFile(config.smb_share, full_file_path, io.BytesIO(backup_file))
+        log.info("Backup nach //" + config.smb_server + '/' + config.smb_share + '/' + full_file_path)
+        try:
+            conn.storeFile(config.smb_share, full_file_path, io.BytesIO(backup_file))
+        except Exception as error:
```

Comment:
Sollte es trotz Pattern in der GUI im Pfad zu einem Zeichen wie z.B. <>| etc. gekommen sein würde das durch die Exception zu einem Errorlog führen: 
Failed to store openwb<>\openWB_backup_2023-10-24_044302.tar.gz on backup: Unable to open file